### PR TITLE
ci: add demo app

### PR DESCRIPTION
**Pull request by splincode**
## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [ ] Feature
-   [ ] Refactoring (no functional changes, no api changes)
-   [x] Other... Please describe:

### CODE REVIEW

File: apps/demo/src/index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
```
Comment (waterplea) [inline apps/demo/src/index.html:1]: Looks like we have 2 `index.html`, is this expected?

File: apps/demo/src/pages/common/common-page.component.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import {ChangeDetectionStrategy, Component} from '@angular/core';
+import README from '../../../../../projects/common/README.md?raw';
+
+@Component({
+    selector: `common-page`,
+    templateUrl: `./common-page.component.html`,
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class CommonPageComponent {
+    content = README;
```
Comment (waterplea) [inline apps/demo/src/pages/common/common-page.component.ts:10]: Is this shown somewhere?

File: apps/demo/src/pages/common/common-page.module.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import {NgModule} from '@angular/core';
+import {CommonPageComponent} from './common-page.component';
+import {RouterModule} from '@angular/router';
+import {CommonModule} from '@angular/common';
+
+@NgModule({
+    declarations: [CommonPageComponent],
+    imports: [
```
Comment (waterplea) [inline apps/demo/src/pages/common/common-page.module.ts:8]: nit: I think imports should go first, this makes more sense semantically imo

File: apps/demo/src/polyfills.ts [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+ *  with the following flag, it will bypass `zone.js` patch for IE/Edge
+ *
+ *  (window as any).__Zone_enable_cross_context_check = true;
+ *
+ */
+
+/***************************************************************************************************
+ * Zone JS is required by default for Angular itself.
+ */
+import '@ng-web-apis/audio/polyfill';
```
Comment (waterplea) [inline apps/demo/src/polyfills.ts:58]: Should we also add universal polyfills?

File: package.json [hunk 5]
```diff
@@ -74,7 +80,6 @@
         "karma-coverage-istanbul-reporter": "3.0.3",
         "karma-jasmine": "4.0.1",
         "karma-jasmine-html-reporter": "1.7.0",
-        "lerna": "6.6.1",
```
Comment (waterplea) [inline package.json:77]: Wasn't there some config file for lerna? Should we remove it too?

File: apps/demo/src/polyfills.ts (hunk 4 above)
Comment (splincode) [reply apps/demo/src/polyfills.ts:58]: image

File: package.json (hunk 5 above)
Comment (splincode) [reply package.json:77]: fixed

File: apps/demo/src/pages/common/common-page.module.ts (hunk 3 above)
Comment (splincode) [reply apps/demo/src/pages/common/common-page.module.ts:7]: fixed

File: apps/demo/src/pages/common/common-page.component.ts (hunk 2 above)
Comment (splincode) [reply apps/demo/src/pages/common/common-page.component.ts]: fixed

File: apps/demo/src/index.html (hunk 1 above)
Comment (splincode) [reply apps/demo/src/index.html:1]: fixed

**Review by waterplea — APPROVED** [review]

**Pull request merged**